algo/src/main/kotlin/com/hartwig/actin/algo/evaluation/treatment/CurrentlyGetsChemoradiotherapyWithSpecificChemotherapyTypeAndMinimumCycles.kt

```diff
@@ -17,7 +17,14 @@ class CurrentlyGetsChemoradiotherapyWithSpecificChemotherapyTypeAndMinimumCycles
 ) : EvaluationFunction {
 
     override fun evaluate(record: PatientRecord): Evaluation {
-        val latestStart = record.oncologicalHistory.maxOfOrNull { LocalDate.of(it.startYear ?: 0, it.startMonth ?: 1, 1) }
+        val latestStartAll = record.oncologicalHistory.maxOfOrNull { LocalDate.of(it.startYear ?: 0, it.startMonth ?: 1, 1) }
+        val latestStart = if (latestStartAll?.let { DateComparison.isAfterDate(it, referenceDate.year, referenceDate.monthValue) } == true){
+            referenceDate
+        }
+        else{
+            latestStartAll
+        }
```

Comment:
I don't see why we'd want to set the latest start date to the reference date. How about:
```suggestion
        val latestStartAll = record.oncologicalHistory.map { LocalDate.of(it.startYear ?: 0, it.startMonth ?: 1, 1) }
            .sortedDescending()
            .firstOrNull { !it.isAfter(referenceDate) }
```